File: 1951-find-the-winner-of-the-circular-game/find-the-winner-of-the-circular-game.java
class Solution {
    public int findTheWinner(int n, int k) {
        ArrayList<Integer> x = new ArrayList<>();
        for (int i = 1; i <= n; i++) {
            x.add(i);
        }
        int index = 0;

        while (x.size() > 1) {
            index = (index + k - 1) % x.size();
            x.remove(index);
        }

        return x.get(0);
    }
}

File: 1951-find-the-winner-of-the-circular-game/find-the-winner-of-the-circular-game.py
class Solution(object):
    def findTheWinner(self, n, k):
        x = list(range(1, n + 1))
        index = 0

        while len(x) > 1:
            index = (index + k - 1) % len(x)
            x.pop(index)
        
        return x[0]
